easybuild/easyconfigs/c/csb/csb-1.2.5-foss-2020b.eb

```diff
@@ -0,0 +1,43 @@
+# Thomas Hoffmann, EMBL Heidelberg, structures-it@embl.de, 2021/05
+easyblock = 'PythonPackage'
+
+name = 'csb'
```

Comment:
The documentation consistently uses CSB for the name - so we should use that here as well.

Reply:
done in fc2f252